### PR TITLE
Unit test For the Math-Magicians Website Pull Request

### PR DESCRIPTION
**Pull request by AleWaweru**
- Did a Unit test using JEST for the operate.js and calculate.js files and also included a snapshot duplicate for both files.

### CODE REVIEW

File: src/__tests__/__snapshots__/calculate.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "operation": "+",
+  "total": 3,
+}
+`;
+
+exports[`calculate should update next number if the button name is a decimal point 1`] = `
+Object {
+  "next": "4.",
+  "operation": "*",
+  "total": 2,
```
Comment (ghost) [inline src/__tests__/__snapshots__/calculate.test.js.snap:15]: - [ ] Good work up to this point but kindly note that it is required in the project requirements that you test all the components. Some of your components like the `calculator`, `navbar`, and the `quote ` have not been tested yet. Kindly add tests to them using Jest snapshots

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ App is working as expected👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/__snapshots__/calculate.test.js.snap (hunk 1 above)
Comment (AleWaweru) [reply src/__tests__/__snapshots__/calculate.test.js.snap:15]: done.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @AleWaweru and @uno36,

✔️ Inclusion of snapshot tests for all components

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**